### DISCUSSION TITLE
combining 2 drawers

### DISCUSSION THREAD

**Discussion by NathanTe** [saadeghi/daisyui#674] [Q&A] [answered]
hey, 

I have a left drawer on 1 pages. 
Now I want to add another drawer to the right side on the parent component. I added the drawer but this seems to fuck with my other drawers styling.
Is this not meant to be combined or did I do something wrong?

**Comment by saadeghi** [comment, marked as answer]
It works if you put the second one inside the `drawer-content` of first one.  
Here's an example: https://play.tailwindcss.com/irwny7uF8M

**Reply by NathanTe** [reply]
I'll check it out, 
Thx for fast reply <3

**Reply by NathanTe** [reply]
@saadeghi 
if the outer drawer has drawer-end it makes the inner one that aswell. 
the example: https://play.tailwindcss.com/iFotLquG49

**Reply by saadeghi** [reply]
@NathanTe Yup. that's a limitation.  
I don't have a solution for it yet but if the inner one has `drawer-end` it works fine.

**Reply by NathanTe** [reply]
@saadeghi 
I would really need it on the right side.
Is there any workaround you can think off a class I can use with important or something?

**Reply by NathanTe** [reply]
if not do you see this getting fixed in the near future?

**Reply by saadeghi** [reply]
Okay I found a quick fix. it works on latest update now (2.13.5)  
https://play.tailwindcss.com/taGIwxGEp5

**Reply by NathanTe** [reply]
i'll try updating the package and let you know

**Reply by NathanTe** [reply]
Works you are the GOAT man. Much love

**Reply by zataara** [reply]
I implemented this strategy into an app I'm building. Do you think there is a workaround to have both drawers open at the same time, one on top of each other?

**Reply by NathanTe** [reply]
So you wanna open the 2nd drawer from your 1st drawer?

**Reply by saadeghi** [reply]
@zataara When a drawer opens, it has a overlay that covers everything underneath. I think if you need multiple sidebars to be visible at the same time, a normal sidebar (using grid or flex) would be fine 🤔 Do we really want to have a lot of complex opening/closing sidebars at the same time?

**Comment by wyysoft** [comment]
The example shows a horizontal scrollbar if DaisyUI version >= 4.4.15 if there are two drawers in one page.
<img width="485" alt="image" src="https://github.com/saadeghi/daisyui/assets/24700314/225792aa-5226-42fa-a3be-d15a9e15c86d">

**Reply by saadeghi** [reply]
@wyysoft Can you please open an issue with more details? Which browser version is this?

**Reply by wyysoft** [reply]
https://github.com/saadeghi/daisyui/discussions/674#discussioncomment-2480930

> It works if you put the second one inside the drawer-content of first one.
Here's an example: https://play.tailwindcss.com/irwny7uF8M

There is a horizontal scrollbar in the example with daisyUI version >= 4.4.15 if two drawers are placed in one page. 
This extra horizontal scrollbar is unexpected.

Browser: Edge latest, Firefox latest both have the problem.

**Reply by wyysoft** [reply]
Please compare these pages:
https://play.tailwindcss.com/2bmLYLiiND?file=config
https://play.tailwindcss.com/UG9iQ4Ax7c?file=config

**Reply by saadeghi** [reply]
@wyysoft I still can't reproduce the issue on my browsers 🤔  
Does the problem still exist in latest version `4.4.17`?

**Reply by wyysoft** [reply]
@saadeghi  I've tried on Mac/Windows, both latest Edge browser, with daisyui v4.4.17. Here is a video.

https://github.com/saadeghi/daisyui/assets/24700314/39221046-2b04-45ae-bee3-ef5487acf755

**Comment by aliqasemzadeh** [comment]
@saadeghi 
I have this problem 
We can not put first drawer in second one.
